src/transformState.js

```diff
@@ -5,7 +5,35 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear': {
+        if (action.type === 'clear') {
```

Comment:
Read more about the switch case. If case 'clear' start to run, your action.type always will be equal 'clear'